test/system/helpers.sh

```diff
@@ -99,3 +99,22 @@ hash_alg_supported() {
         fi
     done
 }
+
+#
+# Verifies that the contexts of a file path provided
+# as the first argument loads as a YAML file.
+#
+function yaml_verify() {
```

Comment:
Nice, you are doing what I suggested :)

But I think this should be one of the first patches in the series, since other patches are duplicating this function in the tests. Also, it seems that Github is messing with the commit order, since I saw in previous patches that you were sourcing this file and using the function...